sign/index.ts

```diff
@@ -13,27 +13,23 @@ const plugin_name = core.getInput('plugin_name');
 async function sign() {
     try {
         await setupPlugin();
-        let output = execSync(`notation plugin ls`, { encoding: 'utf-8' });
-        console.log('notation plugin list output:\n', output);
+        await exec.getExecOutput(`notation plugin ls`);
         const key_id = core.getInput('key_id');
         const plugin_config = core.getInput('plugin_config');
         const target_artifact_ref = core.getInput('target_artifact_reference');
         const signature_format = core.getInput('signature_format');
-        let signOutput;
         if (process.env.NOTATION_EXPERIMENTAL) {
             if (plugin_config) {
-                signOutput = execSync(`notation sign --signature-format ${signature_format} --allow-referrers-api --id ${key_id} --plugin ${plugin_name} --plugin-config=${plugin_config} ${target_artifact_ref}`, { encoding: 'utf-8' });
+                await exec.getExecOutput(`notation sign --signature-format ${signature_format} --allow-referrers-api --id ${key_id} --plugin ${plugin_name} --plugin-config=${plugin_config} ${target_artifact_ref}`);
             } else {
-                signOutput = execSync(`notation sign --signature-format ${signature_format} --allow-referrers-api --id ${key_id} --plugin ${plugin_name} ${target_artifact_ref}`, { encoding: 'utf-8' });
+                await exec.getExecOutput(`notation sign --signature-format ${signature_format} --allow-referrers-api --id ${key_id} --plugin ${plugin_name} ${target_artifact_ref}`);
             }
-            console.log('notation sign output:\n', signOutput);
         } else {
             if (plugin_config) {
-                signOutput = execSync(`notation sign --signature-format ${signature_format} --id ${key_id} --plugin ${plugin_name} --plugin-config=${plugin_config} ${target_artifact_ref}`, { encoding: 'utf-8' });
+                await exec.getExecOutput(`notation sign --signature-format ${signature_format} --id ${key_id} --plugin ${plugin_name} --plugin-config=${plugin_config} ${target_artifact_ref}`);
             } else {
-                signOutput = execSync(`notation sign --signature-format ${signature_format} --id ${key_id} --plugin ${plugin_name} ${target_artifact_ref}`, { encoding: 'utf-8' });
+                await exec.getExecOutput(`notation sign --signature-format ${signature_format} --id ${key_id} --plugin ${plugin_name} ${target_artifact_ref}`);
```

Comment:
Can we leave the first parameter just be `notation`? and leave others to the `args`?